lib/Doctrine/Query.php

```diff
@@ -1447,7 +1450,7 @@ public function getLimitSubquery()
                 // Remove identifier quoting if it exists
                 $e = $this->_tokenizer->bracketExplode($part, ' ');
                 foreach ($e as $f) {
-                    if ($f == 0 || $f % 2 == 0) {
+                    if ($f == 0 || (int) $f % 2 == 0) {
```

Comment:
PHP 8 complained about this usage of running modulo on a string.

I'm not entirely sure what the logic is here (this condition never returns `false` in the tests), so I'm just casting `$f` to an int to keep the behavior the same but making PHP 8 happy.